code/world.dm

```diff
@@ -12,6 +12,9 @@
 
 var/savefile/panicfile
 /world/New()
+	if(fexists("byond-extools.dll") && system_type == MS_WINDOWS)
+		call("byond-extools.dll", "maptick_initialize")()
```

Comment:
>Imagine being this exclusive to Linux

Should try to load the `libbyond-extools.so` on *nix systems.

Reply:
There isn't one.

Reply:
Just make the code check for it and don't provide it, we do this for libvg as well.

Reply:
🤔 
All DLLs and .so's are managed by git, correct?
So why do we need the check if it doesn't exist?
The check can be added in two lines whenever it is available, and again, it probably will never matter unless another host wants to host on Linux, and they will face the same problem. They would also have to be smart enough to build it on Linux, so I'd assume they'd be smart enough to add those two lines.